### ISSUE TITLE
Support for node anchors of arrows

### ISSUE THREAD

**Issue by whisperity** [tikzit/tikzit#138]
TikZ has been supporting specific "coordinate" anchors such as `north` and `west` on nodes when drawing a chart:

```tex
  \draw [style=normal-arrow] (0.east) to (1.west);
```

This way, even a changing-sized node (e.g. a box containing some text) can have proper direction on the arrows in a chart.

However, Tikzit does not support this, even when manually editing the file in the program. The only solution to force the use of this method is to edit the resulting Tikz file in a text editor and never touch it again.

**Comment by novinsh** [comment]
Yes, would be a good feature to have. For some reason, even after editing the source code when opening the tikz file, it resets the modifications for the anchor points!

**Comment by 0xACE** [comment]
It has been a couple of years since i worked on tikzit. but iirc the parser and data structures would need to be adjusted to support parsing of the requested nodes...

I have neglected this machine for a while, turns out obs is not working so ill drop a quick recording

https://user-images.githubusercontent.com/1482942/232899606-24e94e6c-a01d-4389-91c0-d0186a9d7e26.mp4

showcases how you can right click on a line and individually set which direction a arrow should go out from.


and incase you were not referring to a node you can create your own shape and just drag an arrow from there

![cap](https://user-images.githubusercontent.com/1482942/232900031-fcac396f-4e05-4d5b-861d-c01e151edb8b.png)

and iirc if you are drawing more complex shapes it might be worth making that line a single path by selecting those lines and choosing `make path` or w/e it's called.


best of luck
